components/SpotDetails.js

```diff
@@ -1,21 +1,24 @@
 import React from "react";
 import styled from "styled-components";
 
-export default function SpotDetails({
-  name,
-  address,
-  tasks = [],
-  equipment = [],
-}) {
+export default function SpotDetails({ tasks = [], equipment = [] }) {
   return (
     <Details>
-      <Name>{name}</Name>
-      <Address>{address}</Address>
       <Tasks>
         <Title>Tasks:</Title>
+        <ul>
+          {tasks.map((task, index) => (
+            <li key={index}>{task}</li>
+          ))}
```

Comment:
It is better to work with a unique id than with an `index`. [See this article to learn about the usage of keys in React.](https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key)